docs/assets/dev/js/editor/document/readme.md

```diff
@@ -0,0 +1,61 @@
+# Legend
+* Example:
+  ![legend-section-widget-tabs](_images/legend-section-widget-tabs.png)
+- ![#ffd29b](https://via.placeholder.com/15/ffd29b/000000?text=+)  `Section ID` - `98c3441`
+- ![#93ff8b](https://via.placeholder.com/15/93ff8b/000000?text=+) `Column ID` &nbsp;&nbsp;- `18f2ab4`, `4b957d9`
+- ![#43f7ff](https://via.placeholder.com/15/43f7ff/000000?text=+)  `Widget ID` &nbsp;&nbsp;- `5412776`
+
+## Component -- `$e.components.get('document')`
+
+*  **Name**: Document.
+*  **Description**: Document manipulation.
+
+## All **Document** components
+| Component Name                 | Access Component                         | Description
+|--------------------------------|------------------------------------------|-----------------------
+| [Dynamic](dynamic/readme.md)   | `$e.components.get('document/dynamic')`  | Dynamic enable, disable and change.
+| [Elements](elements/readme.md) | `$e.components.get('document/elements')` | Elements manipulation. create, edit, remove, etc...
+| [History](history/readme.md)   | `$e.components.get('document/history')`  | Provides a way of recording\manipulate history of the commands.
+| [Repeater](repeater/readme.md) | `$e.components.get('document/repeater')` | Provides a way communicate with repeater.
+| [Save](save/readme.md)         | `$e.components.get('document/save')`     | Responsible for saving, publish, draft, of the document.
+| [UI](ui/readme.md)             | `$e.components.get('document/ui')`       | Irregular component for handling UI shortcuts.
```

Comment:
"Irregular"? Explain

Reply:
Its Irregular because it's only a wrapper for exist command which are :`copy, delete, duplicate, paste, paste-style` only for keyboard shortcuts. where you want to add it?